spec/abstractops.html

```diff
@@ -986,6 +986,7 @@ <h1>ISO 8601 grammar</h1>
 
       DurationHoursPart :
           DurationWholeHours DurationHoursFraction? HoursDesignator DurationMinutesPart?
+          DurationWholeHours DurationHoursFraction? HoursDesignator DurationSecondsPart?
```

Comment:
Of course, I guess it could be avoided entirely by making the tail mandatory for all but one alternative.
```
      DurationHoursPart :
          DurationWholeHours DurationHoursFraction? HoursDesignator DurationMinutesPart
          DurationWholeHours DurationHoursFraction? HoursDesignator DurationSecondsPart?
      …
      DurationMonthsPart :
          DurationMonths MonthsDesignator DurationWeeksPart
          DurationMonths MonthsDesignator DurationDaysPart?
      …
      DurationYearsPart :
          DurationYears YearsDesignator DurationMonthsPart
          DurationYears YearsDesignator DurationWeeksPart
          DurationYears YearsDesignator DurationDaysPart?
```

Reply:
Good point. In order to avoid ambiguity I've made the change you suggested.